### PR TITLE
docs(.github): update configuration files

### PR DESCRIPTION
**Pull request by lauthieb**
## Changes description

The goal is to align our GitHub different configuration files between `vitamin-(web|android|ios)` stacks.

## Context

This PR is there to enhance the contributor experience and facilitate onboarding for developers.

## Checklist

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch**. Don't request your main!
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks.
- [x] Check PR name, it must follow the https://www.conventionalcommits.org pattern.

## Does this introduce a breaking change?

- No

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/new_component_request.md [hunk 1]
```diff
@@ -4,23 +4,26 @@ about: When a component is ready to dev, create the GitHub issue with this templ
...
 ---
 
-<!--
+<!-- 
+    Before submitting an issue, please consult our [documentation](https://decathlon.design). 
+    Please make sure you are posting an issue pertaining to the Decathlon Design System. 
+
     Note that before opening an issue, you can chat maintainer of this project on channel
-    #vitamin-android in Vitamin Slack. More information here: https://www.decathlon.design/726f8c765/p/195920-contributing
+    #vitamin-web in [Decathlon Design System's Slack(https://join.slack.com/t/decathlon-design/shared_invite/zt-ou0n9qas-n_oamDSVUIqvLqNO1LETJg).
```
Comment (GerardPaligot) [inline .github/ISSUE_TEMPLATE/new_component_request.md:12]: Change to android channel name (maybe link too?)

File: .github/ISSUE_TEMPLATE/new_component_request.md [hunk 2]
```diff
@@ -4,23 +4,26 @@ about: When a component is ready to dev, create the GitHub issue with this templ
...
 
     If you want to know how to contribute to this project, you can check our CONTRIBUTING file:
-    https://github.com/Decathlon/vitamin-android/blob/main/CONTRIBUTING.md
+    https://github.com/Decathlon/vitamin-web/blob/main/CONTRIBUTING.md
```
Comment (GerardPaligot) [inline .github/ISSUE_TEMPLATE/new_component_request.md:15]: Same here

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -37,7 +37,9 @@ If you do not own these types of devices, you can emulate them following [these
 
 ### Reporting bug
 
-The first way to contribute to a project is simply reporting a Bug. If you find anything which is not working well or as expected you can [open an issue](https://github.com/decathlon/vitamin-android/issues/new/choose).
+### Reporting an issue
```
Comment (GerardPaligot) [inline CONTRIBUTING.md:40]: You need to remove the title before too no? (Reporting bug)

Comment (lauthieb) [reply CONTRIBUTING.md:38]: well seen 👀

**Review by lauthieb — COMMENTED** [review]
@GerardPaligot I've fixed your suggestions :)

**Review by GerardPaligot — APPROVED** [review]

**Pull request merged**